tests/glsl/shader-subgroup-builtin-variables.slang

```diff
@@ -0,0 +1,61 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL):  -allow-glsl -stage compute -entry computeMain -target glsl
+//TEST:SIMPLE(filecheck=CHECK_SPV):  -allow-glsl -stage compute -entry computeMain -target spirv
+//TEST:SIMPLE(filecheck=CHECK_HLSL): -allow-glsl -stage compute -entry computeMain -target hlsl
+//TEST:SIMPLE(filecheck=CHECK_CUDA): -allow-glsl -stage compute -entry computeMain -target cuda 
+//TEST:SIMPLE(filecheck=CHECK_CPP):  -allow-glsl -stage compute -entry computeMain -target cpp
+
+//TEST(compute, vulkan):COMPARE_COMPUTE(filecheck-buffer=BUF):-vk -compute -entry computeMain -allow-glsl
+//TEST(compute, vulkan):COMPARE_COMPUTE(filecheck-buffer=BUF):-vk -compute -entry computeMain -allow-glsl -emit-spirv-directly
+#version 430
+
+precision highp float;
+precision highp int;
+
+//TEST_INPUT:ubuffer(data=[2 3 5 7], stride=4):name=inputBuffer
+buffer MyBlockName
+{
+    uint data[];
+} inputBuffer;
+
+//TEST_INPUT:ubuffer(data=[0 0 0 0], stride=4):out,name=outputBuffer
+buffer MyBlockName2 
+{
+    uint data[];
+} outputBuffer;
+
+layout(local_size_x = 32) in;
```

Comment:
You don't seem to need these in this test case.

Reply:
I don't but the tests do need to be changed and to actually test data later, I don't know if we should remove them either (since they will be used at some point)